bind/package.go

```diff
@@ -28,9 +29,17 @@ type Package struct {
 	funcs   []Func
 }
 
+var (
+	mu sync.Mutex
```

Comment:
could you move the declaration of that mutex closer to the declaration of the variable it's guarding access to?
(ie: just above `universe` in the file `bind/symtab.go`)

Reply:
Will do